### DISCUSSION TITLE
Return latest

### DISCUSSION THREAD

**Discussion by mossmoss** [kevinzg/facebook-scraper#147] [Q&A]
What's the easiest way to get the most recent post text? I know a pinned post may or may not be the most recent post, is the pinned post always the first one returned? Will the most recent post always be either the first or second one returned? Of post_id and time, can either be used reliably to determine which post is most recent? If there is any code you can share to achieve it would be very helpful. Thank you!

**Comment by kevinzg** [comment]
I think the best option is to get 2 (or 3 to be extra sure) pages of posts, sort them by post id and take the greater one.
Time might be parsed from a human representation so post id should be better for this.

**Reply by mossmoss** [reply]
Hi Kevin, Thanks for the info. Often times the latest post (or second latest, etc.) will not have any data returned - the data will look like this: 

01951966779618639,,,,2021-02-25 08:28:15,https://scontent-

and the scraper will return None for post text/text. Is there any other way to get this data, with the scraper, or by other means? 

Using: Version: 0.2.17
